src/foam/u2/view/RichChoiceView.js

```diff
@@ -29,7 +29,13 @@ foam.CLASS({
               {
                 heading: 'Users',
                 dao: X.userDAO.orderBy(foam.nanos.auth.User.LEGAL_NAME)
-              }
+              },
+              // Set "disabled: true" to render each object as non-selectable row
+              {
+                disabled: true,
```

Comment:
@TW80000 Why is the User model explicitly referenced in this generic RichChoiceView?
There should be a UserRickChoiceView which extends RichChoiceView and sets the DAO accordingly, or some other reference property.   RichChoiceView should have no explicit knowledge or reference to another model.   This needs to be fixed.